chromium_src/services/network/cookie_settings.cc

```diff
@@ -76,6 +76,7 @@ bool CookieSettings::AnnotateAndMoveUserBlockedEphemeralCookies(
     const GURL& url,
     const net::SiteForCookies& site_for_cookies,
     const url::Origin* top_frame_origin,
+    const net::FirstPartySetMetadata& first_party_set_metadata,
```

Comment:
these should be empty, right?

Reply:
just verifying that this stuff is disabled

Reply:
yes, we have `kFirstPartySets` feature disabled in [chromium_src/content/public/common/content_features.cc](https://github.com/brave/brave-core/blob/master/chromium_src/content/public/common/content_features.cc#L17)